test/circuit/v1/all.js

```diff
@@ -67,55 +67,5 @@ export default {
         createJs([randomWsAddr], factory)
       ])
     }
-  },
-
-  // Below are legacy tests that use js-ipfs as v1 relay
-  // (no tests for go-ipfs as relay v1, because since 0.11 it only supports v2)
-  // FIXME: remove after js-ipfs migrates to v2
```

Comment:
@lidel is this true?  I still see the v1 protocol as supported by `kubo@0.17.x`:

```console
% ipfs id
{
...
	"AgentVersion": "kubo/0.17.0/",
	"ProtocolVersion": "ipfs/0.1.0",
	"Protocols": [
		"/ipfs/bitswap",
		"/ipfs/bitswap/1.0.0",
		"/ipfs/bitswap/1.1.0",
		"/ipfs/bitswap/1.2.0",
		"/ipfs/id/1.0.0",
		"/ipfs/id/push/1.0.0",
		"/ipfs/lan/kad/1.0.0",
		"/ipfs/ping/1.0.0",
		"/libp2p/autonat/1.0.0",
		"/libp2p/circuit/relay/0.1.0",      <------ v1?
		"/libp2p/circuit/relay/0.2.0/stop",
		"/libp2p/dcutr",
		"/p2p/id/delta/1.0.0",
		"/x/"
	]
}
```